index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Gameweek 5 Match Day Live 2022</title>
+    <link rel="stylesheet" href="assets/styles/styles.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <script src="https://use.fontawesome.com/e008c3c639.js"></script>
+    </head>
+    <body class="home-body flex-col">
+        <header class="fnt-l flex-col font-small">
+            <div class="upper-menu flex-col">
+                <ul class="upper-menu-items flex-row clr-w marg-0">
+                    <li><i class="fa fa-facebook" aria-hidden="true"></i></li>
+                    <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <nav class="flex-row">
+                <div class="mobile-menu" id="mobile-menu"><img src="assets/img/mobile_menu.png" class="xxx" alt="mobile-menu"></div>
+                <a href="index.html" class="logo"><img src="assets/img/logo.png" alt="logo"></a>
+                <div class="close hdn"><img src="assets/img/Icon-Cancel.png" alt="close-icon"></div>
+                <ul class="menu hdn">
+                    <li class="menu-item"><a href="about.html">About</a> </li>
+                    <li class="menu-item"><a href="">Program</a> </li>
+                    <li class="menu-item"><a href="">Join</a> </li>
+                    <li class="menu-item"><a href="">Sponsor</a> </li>
+                    <li class="menu-item"><a href="">News</a> </li>
+                    <li class="menu-item active summit-item"><a href="index.html">Match Day</a> </li>
```

Comment:
- [ ] Make sure it should be possible to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too. Kindly add in your navigation bar the link to your home page :+1:

Reply:
This is noted and is going to be done.